xml/article_sap_trento.xml

```diff
@@ -353,6 +353,9 @@ As agreed on https://confluence.suse.com/x/DAEcN on our Trento doc kick off
    oci://registry.suse.com/trento/trento-server \
    --version 0.4.4 \
    --set-file trento-runner.privateKey=<replaceable>PRIVATE_SSH_KEY</replaceable></screen>
+       <para>
+          Please note that the experimental flag is not needed as of helm version 3.8.0.
```

Comment:
Would remove "Please" as it's not recommended in our styleguide. Also would use capital H for helm:

```suggestion
          Note that the experimental flag is not needed as of Helm version 3.8.0.
```